docs/azure_arc_jumpstart/azure_arc_k8s/day2/multi_distributions/calico/_index.md

```diff
@@ -157,33 +165,34 @@ The policy will be set to “Audit” mode to check the configuration of existin
 
   ![Audit details](./16.png)
 
-- Test the network policy enforcement by changing labels in `centos` and `nginx` pods.
+- Test Calico network policy enforcement by adding the `fw-zone` labels to the `centos` and `nginx` pods.
+  > The network policy is set to allow only `trusted` zone connections to the `restricted` zone, and will deny traffic from `dmz` zone. We will label the nginx pod with `fw-zone=restricted`, and test connectivity when changing the centos pod label from `fw-zone=trusted` to `fw-zone=dmz`.
 
-  > The firewall policy only allow `trusted` zone connect to `restricted` zone, and will deny the traffic from `dmz` zone. We will label the nginx pod with `restricted` zone, and test connectivity while changing centos pod label with `trusted` and `dmz` zone.
+- Test Calico network policy enforcement without labels.
 
-  ```shell
-  kubectl -n storefront label pod nginx fw-zone=restricted
-  kubectl -n storefront label pod centos fw-zone=trusted
-  ```
+  ![test result](./22.png)
 
-  ```shell
-  kubectl -n storefront exec -t centos -- sh -c 'curl -m3 -sI http://nginx 2>/dev/null | grep -i http'
-  ```
+- Test Calico network policy enforcement with proper labels.
+
+  ![test result](./23.png)
+
+- Test Calico network policy enforcement with wrong labels.
+
+  ![test result](./24.png)
 
-  ![test result](./22.png)
 
 ## Clean up environment
 
 Complete the following steps to clean up your environment.
 
-- From each Azure Arc-enabled cluster under your resource group, click _Azure Policy_ page, disable the extension of Azure policy under `Onboard to Azure Policy for Azure Arc enabled Kubernetes cluster`
+- From each Azure Arc connected cluster under your resource group, click the _Azure Policy_ page, and disable the extension of Azure policy under `Onboard to Azure Policy for Azure Arc enabled Kubernetes cluster`
```

Comment:
Please update to Azure Arc-enabled Kubernetes cluster

Reply:
done